api/books/finishedlist.ts

```diff
@@ -0,0 +1,19 @@
+import { baseWithAuth } from "api/base"
+
+export const getFinishedList = async () => {
```

Comment:
This looks good but I think the type can be better

```suggestion
export const getFinishedList = async (): Promise<FinishedList[]> => {
```

This will tell typescript that this function will return a promise with `FinishedList[]` type if it is successful.

const bookList = await getFinishedList()
bookList will now have the type FinishedList[]

Reply:
Is it like this? 
![image](https://user-images.githubusercontent.com/91857581/189815710-4716576f-8741-4f33-8a1c-7210c1d6f0d2.png)

so when it gets called it will look like this
![image](https://user-images.githubusercontent.com/91857581/189815763-557592a2-91df-432e-a770-d7998d10c84a.png)

or is it better to make a 'FinishedList' type?

Reply:
Yep this is the expected outcome 👍 
![image](https://user-images.githubusercontent.com/780659/189816436-28206770-ebee-4377-8f65-ffc533bd121c.png)


> better to make a 'FinishedList' type?

What is the list returned by `getFinishedList`? Is it a list of books? Is there any difference between BookList and FinishedList? If the endpoint returns the same structure as `BookList`, then by all means use `BookList[]`

Reply:
yes, it just returns a list of books.

Reply:
I'll go ahead and update the rest of the api calls